integrations/spark/spark-job-dataquality/spark-job-dataquality-app/src/test/java/com/thinkbiganalytics/spark/dataquality/checker/DataQualityCheckerTest.java

```diff
@@ -0,0 +1,109 @@
+package com.thinkbiganalytics.spark.dataquality.checker;
```

Comment:
General comment for tests: In few tests, the message string to be shown when test fails can be made more easy-to-understand. Consider updating the messages.